build.gradle

```diff
@@ -48,6 +75,11 @@ dependencies {
     testImplementation 'com.squareup.okhttp3:mockwebserver:3.11.0'
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.0.0-RC1"
 
+    implementation 'io.grpc:grpc-okhttp:1.28.0' // CURRENT_GRPC_VERSION
+    //implementation 'io.grpc:grpc-protobuf-lite:1.28.0' // CURRENT_GRPC_VERSION
+    implementation 'io.grpc:grpc-protobuf:1.28.0' // CURRENT_GRPC_VERSION
```

Comment:
Ideally, we should be using the `-lite` dependency, but there were many compile issues.  This will need to be addressed later.

Reply:
I need to do further research on the "-lite" functionality.